packages/logger/src/availablity-telemetry.ts

```diff
@@ -0,0 +1,11 @@
+export type AvailabilityTelemetry = {
+    id: string;
+    duration?: string;
+    success?: boolean;
+    runLocation?: string;
+    message?: string;
+    // tslint:disable-next-line: no-any
+    measurements?: any;
```

Comment:
have the any types match the signature as https://github.com/devaradhanm/ApplicationInsights-node.js/blob/0f372d6e3b714b35ce43bd82d7e6646ead07e0fc/Declarations/Contracts/TelemetryTypes/AvailabilityTelemetry.ts